WIP-CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 ### Release notes
+- With the update of the opendistro helm chart you can now decide whether or not you want dedicated deployments for data and client/ingest nodes.
+  By setting `elasticsearch.dataNode.dedicatedPods: false` and `elasticsearch.clientNode.dedicatedPods: false`,
+  The master node statefulset will assume all roles.
+- To get some of the new default values for resource requests on Harbor pods you will first need to remove the resource requests that you have in your Harbor config and then run `ck8s init` to get the new values.
```

Comment:
Should this be here? It looks unrelated :thinking:

Reply:
Yes, the harbor line is old.

Reply:
Yes, a copy/paster error from my side.